client/payment-details/payment-method/afterpay-clearpay/index.tsx

```diff
@@ -30,22 +32,38 @@ const formatPaymentMethodDetails = ( charge ) => {
 };
 
 /**
- * Placeholders to display while loading.
+ * Placeholder object for loading state
  */
-const paymentMethodPlaceholders = {
+const paymentMethodPlaceholders: PaymentMethodDetails = {
```

Comment:
These placeholders appear identical and are shared between both Affirm and Afterpay/Clearpay files. Could we maybe move it to `wcpay/payment-details/types` or some other shared space and import it, once again, to avoid code duplication?